python/src/uagents/envelope.py

```diff
@@ -33,7 +33,7 @@ class Envelope(BaseModel):
     sender: str
     target: str
     session: UUID4
-    schema_digest: str = Field(alias="protocol")
+    schema_digest: str = Field(alias="protocol")  # ???
```

Comment:
This field was initially called `protocol` unfortunately, and this alias was created to preserve backwards compatibility, but this needs to be revisited.

Reply:
Created issue #365